### ISSUE TITLE
TeaSpoon issue or not that is the question...

### ISSUE THREAD

**Issue by teracube** [CortexPE/TeaSpoon#315]
@CortexPE 

PMMP #1544 stable build 
TeaSpoon dev 247 latest build

```
2018-10-09 [22:05:54] [Server thread/INFO]: jamy30125[/7X.XXX.XX.XX:56819] logged out due to Kicked by admin. Reason: Flying is not enabled on this server
2018-10-09 [22:05:54] [Server thread/CRITICAL]: InvalidStateException: "Cannot schedule update on garbage entity pocketmine\Player" (EXCEPTION) in "src/pocketmine/entity/Entity" at line 1403
2018-10-09 [22:05:54] [Server thread/DEBUG]: #0 src/pocketmine/entity/Entity(1784): pocketmine\entity\Entity->scheduleUpdate()
2018-10-09 [22:05:54] [Server thread/DEBUG]: #1 src/pocketmine/Player(2228): pocketmine\entity\Entity->setRotation(double 154.84588623047, double 11.434524536133)
2018-10-09 [22:05:54] [Server thread/DEBUG]: #2 src/pocketmine/network/mcpe/PlayerNetworkSessionAdapter(120): pocketmine\Player->handleMovePlayer(pocketmine\network\mcpe\protocol\MovePlayerPacket object)
2018-10-09 [22:05:54] [Server thread/DEBUG]: #3 src/pocketmine/network/mcpe/protocol/MovePlayerPacket(92): pocketmine\network\mcpe\PlayerNetworkSessionAdapter->handleMovePlayer(pocketmine\network\mcpe\protocol\MovePlayerPacket object)
2018-10-09 [22:05:54] [Server thread/DEBUG]: #4 src/pocketmine/network/mcpe/PlayerNetworkSessionAdapter(92): pocketmine\network\mcpe\protocol\MovePlayerPacket->handle(pocketmine\network\mcpe\PlayerNetworkSessionAdapter object)
2018-10-09 [22:05:54] [Server thread/DEBUG]: #5 src/pocketmine/network/mcpe/protocol/BatchPacket(114): pocketmine\network\mcpe\PlayerNetworkSessionAdapter->handleDataPacket(pocketmine\network\mcpe\protocol\MovePlayerPacket object)
2018-10-09 [22:05:54] [Server thread/DEBUG]: #6 src/pocketmine/network/mcpe/PlayerNetworkSessionAdapter(92): pocketmine\network\mcpe\protocol\BatchPacket->handle(pocketmine\network\mcpe\PlayerNetworkSessionAdapter object)
2018-10-09 [22:05:54] [Server thread/DEBUG]: #7 src/pocketmine/Player(3056): pocketmine\network\mcpe\PlayerNetworkSessionAdapter->handleDataPacket(pocketmine\network\mcpe\protocol\BatchPacket object)
2018-10-09 [22:05:54] [Server thread/DEBUG]: #8 src/pocketmine/network/mcpe/RakLibInterface(161): pocketmine\Player->handleDataPacket(pocketmine\network\mcpe\protocol\BatchPacket object)
2018-10-09 [22:05:54] [Server thread/DEBUG]: #9 vendor/pocketmine/raklib/src/server/ServerHandler(98): pocketmine\network\mcpe\RakLibInterface->handleEncapsulated(string 7X.XXX.XX.XX 56819, raklib\protocol\EncapsulatedPacket object, integer 0)
2018-10-09 [22:05:54] [Server thread/DEBUG]: #10 src/pocketmine/network/mcpe/RakLibInterface(102): raklib\server\ServerHandler->handlePacket()
2018-10-09 [22:05:54] [Server thread/DEBUG]: #11 src/pocketmine/network/Network(94): pocketmine\network\mcpe\RakLibInterface->process()
2018-10-09 [22:05:54] [Server thread/DEBUG]: #12 src/pocketmine/network/mcpe/RakLibInterface(92): pocketmine\network\Network->processInterface(pocketmine\network\mcpe\RakLibInterface object)
2018-10-09 [22:05:54] [Server thread/DEBUG]: #13 vendor/pocketmine/snooze/src/SleeperHandler(120): pocketmine\network\mcpe\RakLibInterface->pocketmine\network\mcpe\{closure}()
2018-10-09 [22:05:54] [Server thread/DEBUG]: #14 vendor/pocketmine/snooze/src/SleeperHandler(82): pocketmine\snooze\SleeperHandler->processNotifications()
2018-10-09 [22:05:54] [Server thread/DEBUG]: #15 src/pocketmine/Server(2259): pocketmine\snooze\SleeperHandler->sleepUntil(double 1539115554.6952)
2018-10-09 [22:05:54] [Server thread/DEBUG]: #16 src/pocketmine/Server(2134): pocketmine\Server->tickProcessor()
2018-10-09 [22:05:54] [Server thread/DEBUG]: #17 src/pocketmine/Server(1700): pocketmine\Server->start()
2018-10-09 [22:05:54] [Server thread/DEBUG]: #18 src/pocketmine/PocketMine(249): pocketmine\Server->__construct(BaseClassLoader object, pocketmine\utils\MainLogger object, string /home/myserver/, string /home/myserver/plugins/)
2018-10-09 [22:05:54] [Server thread/DEBUG]: #19 /home/myserver/PocketMine-MP.phar(1): require(string phar:///home/myserver/PocketMine-MP.phar/src/pocketmine/PocketMine.php)
2018-10-09 [22:05:54] [RakLibServer thread/NOTICE]: Blocked 7X.XXX.XX.XX.27 for 5 seconds
```

**Comment by teracube** [comment]
dktapps said :

"This is a core bug/oversight with the way network batches are handled. It will be fixed in the upcoming 3.2.5 release."

https://forums.pmmp.io/threads/invalidstateexception-cannot-schedule-update-on-garbage-entity-pocketmine-player-exception.6833/#post-58024

so i close this issue

**Comment by CortexPE** [comment]
It wasn't a teaspoon issue anyways xddd
